.gitignore

```diff
@@ -3,3 +3,4 @@ CTestConfig.cmake
 *.*~
 html
 .idea
+CMakeLists_org.txt
```

Comment:
What is this file all about?

Reply:
Sorry, I needed to make changes to CMakeLists.txt, so I keep a copy.  I did not realize that would also commit.  Please ignore all the CMakeLists_xx.txt.  They got included in the commit.

Reply:
OK, the need to be deleted from the PR for sure.